File: dataModuleTester/src/main/java/com/aneonex/bitcoinchecker/tester/volley/UnknownVolleyError.java
package com.aneonex.bitcoinchecker.tester.volley;

import com.android.volley.VolleyError;

public class UnknownVolleyError extends VolleyError {

	private static final long serialVersionUID = -8541129282633613311L;
	
	public UnknownVolleyError(Throwable cause) {
		super(cause);
	}
}

File: dataModuleTester/src/main/java/com/aneonex/bitcoinchecker/tester/volley/UnknownVolleyError.kt
package com.aneonex.bitcoinchecker.tester.volley

import com.android.volley.VolleyError

class UnknownVolleyError(cause: Throwable?) : VolleyError(cause) {
    companion object {
        private const val serialVersionUID = -8541129282633613311L
    }
}